src/tools/clippy/clippy_utils/src/qualify_min_const_fn.rs

```diff
@@ -37,7 +37,7 @@ pub fn is_min_const_fn<'tcx>(tcx: TyCtxt<'tcx>, body: &Body<'tcx>, msrv: &Msrv)
                 | ty::PredicateKind::ConstEvaluatable(..)
                 | ty::PredicateKind::ConstEquate(..)
                 | ty::PredicateKind::TypeWellFormedFromEnv(..) => continue,
-                ty::PredicateKind::AliasEq(..) => panic!("alias eq predicate on function: {predicate:#?}"),
+                ty::PredicateKind::AliasRelate(..) => panic!("alias eq predicate on function: {predicate:#?}"),
```

Comment:
```suggestion
                ty::PredicateKind::AliasRelate(..) => panic!("alias relate predicate on function: {predicate:#?}"),
```

Reply:
there are also two other references to "alias eq" in `rustc_traits/src/chalk/lowering.rs` that ought to be changed

Reply:
those are different alias-eq goals i think? (chalk's version of them, idk tho i'm on my phone)... so i probably won't touch them unless you really think they need to be renamed?

Reply:
oh you're right :woman_facepalming: